apps/lockfile-explorer-web/src/parsing/getPackageFiles.ts

```diff
@@ -0,0 +1,47 @@
+import { IPackageJson } from '../types/IPackageJson';
+
+export const readPnpmfile = async (): Promise<string> => {
+  try {
+    const response = await fetch(`http://localhost:8091/api/pnpmfile`);
+    return await response.text();
+  } catch (e) {
+    console.error('Could not load cjs file: ', e);
+    return 'Missing CJS';
+  }
+};
+
+export const readPackageJSON = async (projectPath: string): Promise<IPackageJson | undefined> => {
+  try {
+    const response = await fetch(`http://localhost:8091/api/package-json`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        projectPath
+      })
+    });
+    return await response.json();
+  } catch (e) {
+    console.error('Could not load package json file: ', e);
+    return undefined;
+  }
+};
+
+export const readParsedCJS = async (projectPath: string): Promise<IPackageJson | undefined> => {
```

Comment:
`readParsedCjs`

Reply:
Also, since the underlying API is `api/package-spec`, maybe this function should be called `getPackageSpec()`?

Reply:
changed